recipe/meta.yaml

```diff
@@ -10,39 +10,53 @@ source:
   sha256: 1baf3d6b049db7cc6cbd7506524cdca45c41e3bfffb24814245fc18f5a8626c2
 
 build:
-  number: 200
+  number: 201
   skip: True  # [win]
   features:
     - blas_{{ variant }}
 
 requirements:
   build:
+    - {{ compiler('c') }}
     - cmake
     - fftw
-    - gdal 2.2.*
     - ghostscript
-    - libnetcdf 4.5.*
-    - hdf5 1.10.1
-    - zlib 1.2.11
+    - gshhg-gmt
+    - dcw-gmt
     - blas 1.1 {{ variant }}
-    - curl >=7.44.0,<8
-    - pcre
-    - openblas 0.2.20|0.2.20.*
+    - {{ pin_compatible('openblas') }}
+    - {{ pin_compatible('gdal') }}
+    - {{ pin_compatible('libnetcdf') }}
+    - {{ pin_compatible('hdf5') }}
+    - {{ pin_compatible('zlib') }}
+    - {{ pin_compatible('curl') }}
+    - {{ pin_compatible('pcre') }}
```

Comment:
Did you use `conda smithy update-cb3`? I don't think we need all these `pin_compatible` here.

Reply:
I didn't. Does it do the conversion automatically? I did this manually. 
OK, should I just use the package name `- curl`?

Reply:
> I didn't. Does it do the conversion automatically?

Yep.

> I did this manually.

Sorry, that was not advertised.

> OK, should I just use the package name - curl?

Try `conda smithy update-cb3` on the original recipe, it should do the right thing 90% of the time.